### ISSUE TITLE
MyDialogManager._OnDownloadedFiles:No Settings Found : could not parse HTTP response header

### ISSUE THREAD

**Issue by astalush** [autodl-community/autodl-irssi#191]
In 2.6.2 version I get this error into the log tab, I can't open the settings of autodlirssi via the rutorrent interface. I have the same error message. 
"Could not check for autodlupdates: Error getting update info: Timed out! Error: Could not parse http response header
Could not check for trackers updates : Error getting update info: Timed out! Error: Could not parse http response header"
![asta](https://user-images.githubusercontent.com/8401863/76664758-bcc05b00-6585-11ea-8641-4641c3bd184b.png)
![asta2](https://user-images.githubusercontent.com/8401863/76664759-bd58f180-6585-11ea-8448-b50e583441d9.png)

**Comment by katiepea** [comment]
same http error, no error with settings/rutorrent
![Screen Shot 2020-03-13 at 4 56 30 PM](https://user-images.githubusercontent.com/12649929/76670032-a7c6d600-654b-11ea-8239-b1d5ba29891f.png)

Net::SSLeay v. is 1.88 , currently cannot get 1.84 to install to test currently, if i downgrade to autodl 2.6.1, works fine and updates

**Comment by madmax76** [comment]
(Net::SSLeay >= 1.88) &&  restart session...

**Comment by katiepea** [comment]
yeah as stated, currently running 1.88, restarting session yields same error

**Comment by madmax76** [comment]
before upgrade to 1.88 it was totally hammering github with authorization requests, this led to temporary block... you might have to wait a while until they like you again... ( < 1hr)

**Comment by katiepea** [comment]
ah i see, is there a cooldown that you know of? i wonder if autochecking will keep me blocked, looking into disabling autocheck

**Comment by madmax76** [comment]
That was < 1 hour here, don't know exactly though.. audo-update is on here...

**Comment by astalush** [comment]
I'm using a docker image (horjulf/rutorrent-autodl) and it installs the latest update (2.6.2). how to downgrade to 2.6.1 if 2.6.2 shows the same error?

**Comment by LOLobo21** [comment]
Same issue with the latest update.

**Comment by astalush** [comment]
Downgraded to 2.6.1, it works in my docker rutorrent.

**Comment by LOLobo21** [comment]
Downgraded. Same issue.

` Irssi v1.2.2 - http://www.irssi.org                                                                                    
10:43 -!- Irssi: You are running autodl-irssi v2.6.1 | autodl-trackers v282
10:43 -!- Irssi: Bugs and Requests: https://github.com/autodl-community
10:43 -!- Irssi: Discussion: #autodl-community on irc.p2p-network.net
10:43 -!- Irssi: Successfully loaded tracker files
10:43 -!- Irssi: Reading configuration files
10:43 -!- Irssi: Configuration files loaded
10:43 -!- Irssi: Loaded 1000 releases from history file.
10:43 -!- Irssi:  ERROR:  Could not check for trackers updates: Error getting update info: HTTP/1.1 403 Forbidden
10:43 -!- Irssi:  ERROR:  Could not check for autodl updates: Error getting update info: HTTP/1.1 403 Forbidden`

Also, it seems the plugin re-downloaded the v2.6.2 update and is receiving the same error.

**Comment by sorentorp** [comment]
Same issue here. Anyone looking into this?

**Comment by lun4r** [comment]
I am having a similar issue
autodl-irssi v.2.6.2 /autodl-trackers v282

**Comment by LOLobo21** [comment]
The plugin is now working. Edited the config file to turn off auto-update AND downgraded to 2.6.0.

**Comment by ThisGuyNeedsABeer** [comment]
Getting the same errors with autodl-irssi 2.6.2/rTorrent 0.9.6/ruTorrent 3.9. How would one download the autodl-irssi version of a docker image (xeroxmalf/rutorrent-autodl-irssi), specifically.

**Comment by astalush** [comment]
I’ve stopped the container, replaced all autodlirssi config files with the ones from 2.6.0 an restarted the container. It’s working. Also stopped auto update.

**Comment by ThisGuyNeedsABeer** [comment]
I resolved using the LS.io container (as I was using), but with the following repo: (horjulf/rutorrent-autodl:develop)
